api/REST-API.md

```diff
@@ -1,191 +1,596 @@
 # REST API
 
-### Projected Time
+## Projected Time
 
 About 2.5 hrs
 
 - Lesson: 30 min
 - Independent Practice: 120 min
 
-### Prerequisites
+## Prerequisites
 
 Here are links to lessons that should be completed before this lesson:
 
 - [API/JSON Lesson](/api/apis-and-json.md)
 
-### Motivation
```

Comment:
Do you think it would be worth mentioning that “what is an api” and “what is rest” are common interview questions? The motivation is essentially “why you should pay attention “

Reply:
Absolutely! Great idea.

see [1c62c05](https://github.com/Techtonica/curriculum/pull/1410/commits/1c62c05a5fef0b7f4e4a3eba5ae6459a5019094c)